modules/hmftools/1.0/config/default.yaml

```diff
@@ -0,0 +1,106 @@
+lcr-modules:
+    
+    hmftools:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} 
+            # Note: Only outputs of running Strelka in paired/matched normal mode are accepted
+            # Must be in bgzip with `.vcf.gz` extension. 
+            strelka_vcf: "__UPDATE__" 
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            # Note: These are output by the gripss somatic filtering step of the gridss module
+            gridss_somatic: "__UPDATE__" # Output of GRIPSS
+            gridss_somatic_tbi: "__UPDATE"
+            gridss_somatic_filtered: "__UPDATE__" # Filtered output of GRIPSS
+            gridss_somatic_filtered_tbi: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        references: 
+            linx_db: "/projects/rmorin/reference/hmftools-references/linx"
+
+        switches: 
```

Comment:
For non-GSC users can you document where these files can be obtained or provide URLs as comments?